source/administration/image-proxy.rst

```diff
@@ -3,7 +3,7 @@
 Image Proxy
 ================================
 
-Using image proxies means that users make only secure requests. By allowing image requests to go straight to third-party
+Using image proxies means that users make only secure requests for Markdown embedded images. By allowing image requests to go straight to third-party
```

Comment:
We could change "users" to "clients" to make this a bit clearer, but the changes to this line are incorrect. Clients use the image proxy for all image requests which is what this document is talking about, but the issue linked below refers specifically to server requests.

Reply:
Okay how would you update then to accurately document linked bug? It sounds like while the client may get the image from the Mattermost server the sever is not using the image proxy but rather grabbing and serving image directly.

Reply:
The way it currently works in more detail is that:
1. When a Markdown image or plain text link is posted, the server hits the URL directly to see what type of content it is
2. The server tells the clients what type of content is in the link
3. The client requests the image from the server and the server redirects the request through the proxy if necessary.

The first request is the problem there since it's before the server knows if it can use the image proxy. Maybe we can clarify that the server needs to make that first request before it confirms the data?

It's been a while since I've last looked at that bug, and I'm not quite sure if it's a bug. It might not be ideal, but when we've thought about improving it in the past (such as making non-image requests through the image proxy), it's come down to basically replacing the image proxy with a full proxy.

Reply:
Updated accordingly. Please feel free to edit as you see fit.